source/routing/isochrone.cpp

```diff
@@ -200,10 +233,11 @@ type::MultiPolygon build_single_isochrone(RAPTOR& raptor,
             circles_classed.push_back(to_add);
         }
     }
-    boost::sort(circles_classed,
+    std::vector<InfoCircle> circles_check = delete_useless_circle(circles_classed, speed);
```

Comment:
it seems that `circles_classed` is consumed by `delete_useless_circle`, so you can do `auto circles_check = delete_useless_circle(std::move(circles_classed), speed);`